cspm/rn/prisma-cloud-compute-release-information/look-ahead-planned-updates-prisma-cloud-compute.adoc

```diff
@@ -1,11 +1,81 @@
 [#idbcabf073-287c-4563-9c1f-382e65422ff9]
-== Look Ahead—Planned Updates on Prisma Cloud Compute
+== Look Ahead — Planned Updates on Prisma Cloud Compute
 
 // Review any deprecation notices and new features planned in the next Prisma Cloud Compute release.
 
 See xref:prisma-cloud-compute-release-information.adoc#id79d9af81-3080-471d-9cd1-afe25c775be3[Prisma Cloud Compute Release Information] for the latest features of the host, container, and serverless capabilities that are available on the *Compute* tab on Prisma Cloud.
 
-Currently there are no previews or announcements for updates.
+The details and functionality listed below are a preview of what is planned in the next Compute update planned for June 16, 2023; the changes listed herein and the actual release date is subject to change.
 
-//Note that the details and functionality listed below are a preview of what is planned in the next Compute update planned for May 23, 2023; the changes listed herein and the actual release date is subject to change.
+=== Deprecation Notices
 
+Review the list of all deprecation notices in Prisma Cloud Compute.
+
+
+=== Breaking Changes
+
+Be aware of the following breaking changes with the upgrade to 30.02.XXX:
+
+=== Changes in Existing Behavior
+
+* On pre-existing compliance rules, newly-added compliance checks are set to ignore, regardless of severity.
+* Vulnerability discovery dates are no longer updated upon upgrade.
++
+When your Compute console is upgraded from 21.08 to 22.01, the vulnerability discovery dates will be updated. However, on subsequent upgrades from 22.01, vulnerability discovery dates will be preserved.
+
+//CWP-47729
+* Fixed an issue with the Defenders and agentless scans detecting an incorrect Kubernetes version.
```

Comment:
@rcaballeromx This is a bug fix and should be only included in the RN under "Addressed Issues".

Reply:
Done @divyabhushan